engines/draci/draci.cpp

```diff
@@ -98,6 +150,7 @@ DraciEngine::DraciEngine(OSystem *syst, const ADGameDescription *gameDesc)
 	_dubbingArchive = nullptr;
 	_showWalkingMap = 0;
 	_pauseStartTime = 0;
+	_language = gameDesc->language;
```

Comment:
The standard approach is slightly different.

1. Add to metanegine.cpp, the following method:
```C++
Common::Language DraciEngine::getLanguage() const {
        return _gameDescription->language;
}
```
2. Use everywhere `getLanguage()` call